packages/ses/src/lockdown-shim.js

```diff
@@ -61,6 +61,11 @@ export function repairIntrinsics(
   // Thus, all present options must agree with first options.
   // Reconstructing `option` here also ensures that it is a well
   // behaved record, with only own data properties.
+  //
+  // The `'overrideTaming'` is not a safety issue. Rather it is a tradeoff
+  // between code compatibility, which is better with the `'moderate'`
+  // setting, and tool compatibility, which is better with the `'min'`
+  // setting.
```

Comment:
I could use guidance on when to use `min` and when to use `moderate` here, or how to thread a variable that decides automatically.

Reply:
Done. See https://github.com/Agoric/SES-shim/blob/override_safety/packages/ses/README.md#enabling-override-by-assignment section Enabling Override by Assignment.

Ready for review. PTAL. Thanks.